fog/ledger/enclave/impl/src/lib.rs

```diff
@@ -161,7 +172,9 @@ where
         })?;
 
         let mut resp = CheckKeyImagesResponse {
-            num_blocks: untrusted_key_image_query_response.highest_processed_block_count,
+            num_blocks: untrusted_key_image_query_response
+                .processed_block_range
+                .end_block,
```

Comment:
Just confirming, `end_block` is zero-indexed, so if `end_block` is 5 on a range of [0,5], there's actually 6 blocks in the response?

Reply:
End block is zero indexed, however it is exclusive, https://github.com/mobilecoinfoundation/mobilecoin/blob/21d3af4d7122bb4bfeae7ff4cff3d78b9b1a323b/fog/types/src/common.rs#L12
(Also I have no idea why that's in fog/types...)
so [0, 5) would be block indices 0, 1, 2, 3, 4

Reply:
Ah perfect ty!

Reply:
added code comment